frontend/packages/shared/src/features/dataModelling/DataModelling.tsx

```diff
@@ -131,8 +131,8 @@ export function DataModelling({
   const handleHideIntroPageButtonClick = () =>
     setHideIntroPage(setLocalStorageItem('hideIntroPage', true));
 
-  const [editMode, setEditMode] = useState(() => getLocalStorageItem('editMode'));
-  const toggleEditMode = () => setEditMode(setLocalStorageItem('editMode', !editMode));
+  const [editMode] = useState(() => getLocalStorageItem('editMode'));
+
```

Comment:
This can probably be removed since we don't store/use it anymore
```suggestion
```

Reply:
done